<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesInCategory</class>
 <widget class="QMainWindow" name="NotesInCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>82</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>82</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(27, 43, 66);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_36">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="NCHomeButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/House4.png</normaloff>../PNG/House4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="NCCategoryComBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>169</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>187</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic Medium</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
	color: rgb(27, 43, 66);
    border: 2px solid  rgb(27, 43, 66);
    border-radius: 10px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FCFCFC, stop: 0.4 #FAFAFA,
                                 stop: 0.5 #F5F5F5, stop: 1.0 #F2F2F2);
}


QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #F2F2F2, stop: 0.4 #F5F5F5,
                                stop: 0.5 #FAFAFA, stop: 1.0 #FCFCFC);
}

QComboBox:on { 
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
	border-bottom-color: rgb(27, 43, 66);
	border-left-color: rgb(212, 212, 214);
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { 
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: white;
}
</string>
           </property>
           <item>
            <property name="text">
             <string>Select category</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NCSearchLEdit">
           <property name="minimumSize">
            <size>
             <width>187</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>187</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic Medium</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: white;
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="NCSearchButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/Magnifier4.png</normaloff>../PNG/Magnifier4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="NCUserButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PNG/profile4.png</normaloff>../PNG/profile4.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QScrollArea" name="NCScrollArea">
          <property name="styleSheet">
           <string notr="true">background: white;
border-style:solid; 
border-width: 0.5px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(27, 43, 66);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>896</width>
             <height>530</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="NCCreateNoteButton">
        <property name="minimumSize">
         <size>
          <width>257</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>257</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(27, 43, 66);
border-style:solid; 
border-width: 2px; 
border-radius: 10px;
border-color: rgb(27, 43, 66); 
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Create Note</string>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
